Case Studies (#718)
withastro/astro.build: src/content/caseStudies/contenda-share-pear/index.mdx

---
title: "Content for Everyone"

> description: "Building Public-Sharing of Blog Drafts in 1 Month"

Reviewer: ```suggestion
description: "Building Public-Sharing of Blog Drafts in One Month"
```

Reviewer 2: This still needs fixing to read “One” right?

Reviewer: Yup, bumping so Darius sees and commits! I thought it got lost in git history or something.

<div class="text-white space-y-8">
	<div class="text-2xl leading-normal tracking-normal text-astro-gray-100">
		[Contenda](https://contenda.co/), an artificial intelligence company that came onto the scene in
		2021, has been building towards the ambitious goal of making content for everyone. Their vision
		starts with being a catalyst for developer advocates by making repurposing content easy for a
		variety of mediums.
	</div>

Developer advocates are the bridge between software companies and their developer users, often creating content to help educate and raise awareness in the community about particular features. Contenda makes it possible to take, for example, a video tutorial and turning it into a blog. This saves time and can even make it easier for a cross-functional team to collaborate.

Grady Salzman, software engineer, says, “Depending on your team size, Contenda could enable others to make content for you. You could have someone who is spending all their time and effort on recording videos. Someone else could take those videos and write a blog post in their own thoughts, tone, and voice. If you’re a smaller team, it could enable you to release faster.”

> However, for all the ways Contenda was theoretically supposed to make collaboration and content creation easier, it needed to cross one practical hurdle: being able to share blog drafts publicly with other people. Contenda solved this by choosing Astro and Netlify as their solution.

Reviewer: ```suggestion
When Contenda needed to share draft blog posts publicly with other people, they turned to Astro and Netlify as their solution. This combination of technologies let them stay laser focused on their goal of making collaboration and content creation easy.
```

</div>

## Challenges

Contenda is a team of less than 10 people. They often need to share blogs with each other in order to address pain points and conduct reviews but they couldn’t do these things without sharing their emails and API keys with one another. This also posed a security problem for customers that they knew they had to solve eventually. What would eventually be known and open-sourced as Share Pear was created to resolve both issues for internal and external users.

## Requirements

Share Pear needed to be built without requiring backend changes since Contenda did not want to overload themselves by managing additional complexity. Ideally, it also needed to be cost-effective but preferably free. Fortunately for the team, they were able to assess and select Astro as their solution all in a day’s time thanks to quick work from Cassidy Williams.

> Cassidy is Contenda’s CTO. Having previously worked at Netlify plus being an active developer in the Astro community, she saw an opportunity to use Astro with Netlify’s on-demand builders. In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, you can choose to have React or HTML and CSS. The latter enables non-developer team members, such as a product designer, to make changes without requiring developer intervention.

Reviewer: ```suggestion
Cassidy is Contenda’s CTO. Having previously worked at Netlify plus being an active developer in the Astro community, she saw an opportunity to use Astro with Netlify’s on-demand builders. In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, popular JavaScript frameworks like React are available, but optional. A site can be built using just HTML, CSS and, if necessary, vanilla JavaScript. This enables team members with less web development experience, such as product designers and content authors, to edit and maintain the site often without any developer intervention required.
```

Contenda also knew they had to use Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.

## Solution and Implementation

> At first, it was hard for the team to get started because there were barely any written tutorials about using Astro together with Netlify’s on-demand builders let alone just the latter. Their team ended up going into Astro’s Discord to enlist the help of the core team and the community. From there, building was fast for the team at Contenda because the developer experience was good.

Reviewer: ```suggestion
Contenda found success in Astro’s Discord by enlisting the help of the core team and the community. From there, building was fast because the developer experience was good.
```

---
title: "Content for Everyone"

> description: "Building Public-Sharing of Blog Drafts in 1 Month"

Reviewer: ```suggestion
description: "How Astro Helped Contenda Build a Content-Sharing Platform for Everyone"
```

seo:
  title: "Content for Everyone: Building Public Sharing of Blog Posts in One Month"

>   description: "How an AI content company used Astro and Netlify's on-demand builders to enable users to share their generated blog posts with each other."

Reviewer: ```suggestion
  description: "How an AI content company used Astro and Netlify's on-demand builders to enable users to share their generated blog posts with each other - all in one month."
```

Cassidy (Contenda CTO) saw an opportunity to pair Astro with Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.

## Solution and Implementation

Contenda found success in Astro’s developer community. The team was able to enlist the help of the core team and active contributors in [Astro's community Discord server](https://astro.build/chat). From there, building Share Pear moved quickly.

<Blockquote>
	<Fragment slot="quote">
		This was my first time using Astro. I would say, getting started with Astro was simpler than
		other frameworks. It felt like the onboarding time was way smoother and especially the docs are
		so up-to-date that it’s a 1-to-1 match with your code sometimes.
	</Fragment>
	<Fragment slot="caption">— Grady Salzman, Software Engineer @ Contenda</Fragment>
</Blockquote>

<BlogContentImage src={screenshot1} />

## Results

> [Share Pear](https://github.com/Contenda-Team/share-pear) has been online for about a month and is used by both the open-source community and customers of Contenda that [use their app](https://signup.contenda.co/). To-date, 30% of their users share blogs using the Share Pear method and this feature has become one of the top 3 most used on the platform.

Reviewer: ```suggestion
[Share Pear](https://github.com/Contenda-Team/share-pear) has been online for about a month and is used by both the open-source community and customers of Contenda that [use their app](https://signup.contenda.co/). At time of writing, 30% of their users share blogs using the Share Pear method and this feature has become one of the top 3 most used on the platform.
```

---
title: "Content for Everyone"

> description: "How Astro Helped Contenda Build a Content-Sharing Platform for Everyone"

Reviewer: ```suggestion
description: "How Astro Helped Contenda Build a Content-Sharing Platform in One Month"
```

seo:

>   title: "Content for Everyone: How Astro Helped Contenda Build a Content-Sharing Platform for Everyone"

Reviewer: ```suggestion
  title: "Content for Everyone: How Astro Helped Contenda Build a Content-Sharing Platform in One Month"
```

  description: "How an AI content company used Astro and Netlify's on-demand builders to enable users to share their generated blog posts with each other - all in one month."

> publishDate: "June 20, 2023"

Reviewer: ```suggestion
publishDate: "June 29, 2023"
```

---

import shareFeature from "./share_feature.webp"
import results from "./results.webp"
import videosToContent from "./videos-to-content.webp"
import BlogContentImage from "~/components/BlogContentImage.astro"
import ImageFigure from "~/components/ImageFigure.astro"
import Blockquote from "~/components/Blockquote.astro"

<div class="text-white space-y-8">
	<div class="text-2xl leading-normal tracking-normal text-astro-gray-100">
		[Contenda](https://contenda.co/), an artificial intelligence company that came onto the scene in
		2021, has been building towards the ambitious goal of making content for everyone. Their vision
		starts with being a catalyst for developer advocates by making repurposing content easy for a
		variety of mediums.
	</div>

Developer advocates are the bridge between software companies and their developer users, often creating content to help educate and raise awareness in the community about particular features. Contenda makes it possible to take, for example, a video tutorial and turning it into a blog. This saves time and can even make it easier for a cross-functional team to collaborate.

> Grady Salzman, software engineer, says, “Depending on your team size, Contenda could enable others to make content for you. You could have someone who is spending all their time and effort on recording videos. Someone else could take those videos and write a blog post in their own thoughts, tone, and voice. If you’re a smaller team, it could enable you to release faster.”

Reviewer: lmao "dramatic". yeah it's to plug them. could be removed! I think we already have another Grady quote as well (that one's about Astro though, which is muy bien).

> Grady Salzman, software engineer, says, “Depending on your team size, Contenda could enable others to make content for you. You could have someone who is spending all their time and effort on recording videos. Someone else could take those videos and write a blog post in their own thoughts, tone, and voice. If you’re a smaller team, it could enable you to release faster.”

Reviewer: ```suggestion
```

When Contenda needed to share draft blog posts publicly with other people, they turned to Astro and Netlify as their solution. This combination of technologies let them stay laser focused on their goal of making collaboration and content creation easy.

</div>

<div class="my-8">
	<ImageFigure caption="Take any video and turn it into effective written content with Contenda.">
		<BlogContentImage src={videosToContent} />
	</ImageFigure>
</div>

## Challenges

Contenda is a team of less than 10 people. They often need to share blogs with each other in order to address pain points and conduct reviews but they couldn’t do these things without sharing their emails and API keys with one another. This also posed a security problem for customers that they knew they had to solve eventually. To resolve these issues for internal and external users, Contenda created [Share Pear](https://github.com/Contenda-Team/share-pear), an open-source tool for quickly sharing Markdown blog post previews.

## Requirements

> [Share Pear](https://github.com/Contenda-Team/share-pear) needed to be built without requiring backend changes since Contenda did not want to overload themselves by managing additional complexity. Ideally, it also needed to be cost-effective but preferably free. Fortunately for the team, they were able to assess and select Astro as their solution all in a day’s time thanks to quick work from Cassidy Williams.

Reviewer: Approved ✅

> Cassidy is Contenda’s CTO. Having previously worked at Netlify plus being an active developer in the Astro community, she saw an opportunity to use Astro with Netlify’s on-demand builders. In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, popular JavaScript frameworks like React are available, but optional. A site can be built using just HTML, CSS and, if necessary, vanilla JavaScript. This enables team members with less web development experience, such as product designers and content authors, to edit and maintain the site often without any developer intervention required.

Reviewer 3: ```suggestion
In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, popular JavaScript frameworks like React are available, but optional. A site can be built using just HTML, CSS and, if necessary, vanilla JavaScript. This enables team members with less web development experience, such as product designers and content authors, to edit and maintain the site often without any developer intervention required.
```

Reviewer: Approved ✅

> Contenda also knew they had to use Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.

Reviewer 3: ```suggestion
Cassidy (Contenda CTO) saw an opportunity to pair Astro with Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.
```

Reviewer: Approved ✅

## Solution and Implementation

> Contenda found success in Astro’s Discord by enlisting the help of the core team and the community. From there, building was fast because the developer experience was good.

Reviewer 3: ```suggestion
Contenda found success in Astro’s developer community. The team was able to enlist the help of the core team and active contributors in [Astro's community Discord server](https://astro.build/chat). From there, building Share Pear moved quickly.
```

Reviewer: Approved ✅

> Cassidy is Contenda’s CTO. Having previously worked at Netlify plus being an active developer in the Astro community, she saw an opportunity to use Astro with Netlify’s on-demand builders. In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, you can choose to have React or HTML and CSS. The latter enables non-developer team members, such as a product designer, to make changes without requiring developer intervention.

Reviewer 3: duplicate paragraph of above?
